components/CollectiveCover.js

```diff
@@ -147,9 +151,50 @@ class CollectiveCover extends React.Component {
                 />
                 , &nbsp;
                 <FormattedTime value={props.collective.endsAt} timeZone={props.collective.timezone} />
-                &nbsp; - &nbsp;
+                &nbsp;{' '}
+                {moment()
+                  .tz(props.collective.timezone)
+                  .zoneAbbr()}{' '}
+                (Event Time)
               </React.Fragment>
             )}
+            <br />
+            <em>
+              {this.checkTimeDiff() && props.collective.startsAt && (
+                <React.Fragment>
+                  <FormattedDate
+                    value={props.collective.startsAt}
+                    timeZone={momentTimezone.tz.guess()}
+                    weekday="short"
+                    day="numeric"
+                    month="long"
+                  />
+                  , &nbsp;
+                  <FormattedTime value={props.collective.startsAt} timeZone={momentTimezone.tz.guess()} />
+                  &nbsp; - &nbsp;
+                </React.Fragment>
+              )}
+
+              {this.checkTimeDiff() && props.collective.endsAt && (
+                <React.Fragment>
+                  <FormattedDate
+                    value={props.collective.endsAt}
+                    timeZone={momentTimezone.tz.guess()}
+                    weekday="short"
+                    day="numeric"
+                    month="long"
+                    year="numeric"
+                  />
+                  , &nbsp;
+                  <FormattedTime value={props.collective.endsAt} timeZone={momentTimezone.tz.guess()} />
+                  &nbsp;{' '}
+                  {moment()
+                    .tz(momentTimezone.tz.guess())
+                    .zoneAbbr()}{' '}
+                  (Your Time)
+                </React.Fragment>
+              )}
+            </em>
```

Comment:
Would be great to not re-compute this value two times with `this.checkTimeDiff()`. Can we wrap the entire `em` block in a `{this.checkTimeDiff() && (....)}` condition rather than checking once for startsAt and another one for endsAt?